src/main/java/org/dependencytrack/resources/v1/AccessControlResource.java

```diff
@@ -160,4 +161,35 @@ public Response deleteMapping(
             }
         }
     }
+
+    @GET
+    @Path("/notassignedprojects/{teamUuid}")
```

Comment:
Could this be added to the `/api/v1/project` endpoint instead? We already have a few query parameters there that allow for filtering of results:

https://github.com/DependencyTrack/dependency-track/blob/2f16cae2c36fda9ac6054fe0ea5c3a4130e42bfa/src/main/java/org/dependencytrack/resources/v1/ProjectResource.java#L88-L98